src/site/stages/build/process-cms-exports/schemas/common/generic-nested-string.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
     properties: {
       value: { type: 'string' },
     },
+    required: ['value'],
```

Comment:
I'm doing this so that the schema validation will let us know if `value` isn't present.